platform-sdk/swirlds-virtualmap/build.gradle.kts

```diff
@@ -15,23 +15,24 @@
  */
 
 plugins {
-  id("com.swirlds.platform.conventions")
-  id("com.swirlds.platform.library")
-  id("com.swirlds.platform.maven-publish")
-  id("com.swirlds.platform.benchmark-conventions")
+    id("com.swirlds.platform.conventions")
+    id("com.swirlds.platform.library")
```

Comment:
intentional?

Reply:
I assume that files were never touched after we introduced the new formatting and spotless.